src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplication.groovy

```diff
@@ -18,6 +18,17 @@ package com.bmuschko.gradle.docker
 class DockerJavaApplication {
     String baseImage = 'java'
     String maintainer = System.getProperty('user.name')
+    @Deprecated
     Integer port = 8080
+    Integer[] ports = []
     String tag
+
+    Integer[] getPorts() {
```

Comment:
@Salmondx on second thought I think your point out weighs the points I was about to make ;) Lets leave your implementation. The only thing I would change is that when you're returning `port`do something like `return [port]` as what you're doing is returning the single instance instead of the expected array.